CMSgov/mint-app
# [NOREF] LD string flag and conditional date alert

# NOREF



## Description


- Added LD string flag `changeHistoryReleaseDate`
- Added gql field on model plan for `createdDts`
- Added conditional alert to render only if model plan was created before CH release
- Rendered out LD string value

## How to test this change

- Verify CH alert is rendered for a model plan with the date uplled from LD
- Change the LD string value for `changeHistoryReleaseDate` to a date in the past
  - https://app.launchdarkly.com/projects/mint/flags/changeHistoryReleaseDate/variations?env=local&env=dev&env=test&env=impl&env=production&selected-env=local
- Verify the date alert no longer shows

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
